tests/integration/sqlalchemy/test_event_outbox.py

```
@@ -52,19 +61,38 @@ async def handle(self, request: OutboxRequest) -> None:
+
+
 class TestOutbox:
-    async def test_outbox_add_3_event_positive(self, session):
+
+    @pytest.fixture(autouse=True)
+    async def setup_db(self, engine):
+        async with engine.begin() as conn:
+            await conn.run_sync(Base.metadata.create_all)
+
+    @pytest.fixture(autouse=True)
+    async def cleanup_table(self, session, setup_db):
```

Ruff (0.14.14)

[warning] Unused method argument: `setup_db`

(ARG002)